scipy/signal/tests/test_filter_design.py

```diff
@@ -893,6 +893,14 @@ def test_nyquist(self):
         assert_array_almost_equal(w1, w2)
         assert_array_almost_equal(h1, h2)
 
+    # https://github.com/scipy/scipy/issues/17289
+    # https://github.com/scipy/scipy/issues/15273
+    def test_17289(self):
+        d = [0,1]
+        w, Drfft = freqz(d, worN=32, include_nyquist=True)
+        _, Dpoly = freqz(d, worN=w)
+        assert_array_almost_equal(Drfft, Dpoly)
```

Comment:
New tests should use `assert_allclose` instead of `assert_array_almost_equal`

Reply:
fixed